helix-core/src/main/java/org/apache/helix/messaging/handling/HelixTaskExecutor.java

```diff
@@ -225,14 +249,12 @@ public void registerMessageHandlerFactory(String type, MessageHandlerFactory fac
         _monitor.createExecutorMonitor(type, newPool);
         return newPool;
       });
-      LOG.info(
-          "Registered message handler factory for type: " + type + ", poolSize: " + threadpoolSize
-              + ", factory: " + factory + ", pool: " + _executorMap.get(type));
+      LOG.info("Registered message handler factory for type: " + type + ", poolSize: " + threadpoolSize + ", factory: "
```

Comment:
nit: I would suggest using [parameterized logging](http://www.slf4j.org/faq.html#logging_performance). String + String adds a lot cost when constructing the message parameter.

Reply:
Sure.